script/npm_lint_plugins.js

```diff
@@ -1,20 +1,84 @@
 #!/usr/bin/env node
 /* eslint-disable no-var */
+/* eslint-disable no-console */
 
 var fs = require('fs');
 
-var childProcess = require('child_process');
-var { packageJsonDirs } = require('./plugin_webpack_directories');
+const { spawn, execSync } = require('child_process');
+const path = require('path');
+
+var { packageJsonDirsObject } = require('./plugin_webpack_directories');
+
+var dirs = packageJsonDirsObject();
+var dirsKeys = Object.keys(dirs);
+var passedArgs = process.argv.slice(2);
 
 function pluginDefinesLint(pluginPath) {
   var packageData = JSON.parse(fs.readFileSync(`${pluginPath}/package.json`));
 
   return packageData.scripts && packageData.scripts.lint;
 }
 
-packageJsonDirs().forEach(pluginPath => {
-  if (pluginDefinesLint(pluginPath)) {
-    childProcess.spawn('npm', ['run', 'lint'], {
+if (passedArgs[0] && passedArgs[0][0] !== '-') {
+  // if the argument is --fix and not a plugin name npm lint:plugins katello --fix
+  dirsKeys = dirsKeys.filter(dir => dir.endsWith(passedArgs[0]));
+  passedArgs.shift();
+}
+try {
+  const scriptPath = path.join(__dirname, 'lint', 'link-eslint-plugin.js');
+  execSync(`node ${scriptPath}`, { stdio: 'inherit' });
+} catch (error) {
+  console.error(`Error: ${error.message}`);
+}
+
+const packageJsonDirectories = [
+  './',
+  './node_modules/@theforeman/vendor-core/',
+];
+dirsKeys.forEach(dirsKey => {
+  const pluginPath = dirs[dirsKey];
+  const pluginLintScript = pluginDefinesLint(pluginPath);
```

Comment:
Shouldn't we do the same thing as we do with lint?
If a plugin defines a `test` script - run it, otherwise use this one?